corpus/accounts/models.py

```diff
@@ -135,13 +136,16 @@ class ExecutiveMember(models.Model):
         blank=True, null=True, verbose_name="Linkedin Profile URL"
     )
 
+    profile_picture = models.ImageField(
+        blank=True, null=True, upload_to="execmember/profile_picture"
```

Comment:
```suggestion
        blank=True, null=True, upload_to="accounts/executivemember/profile_picture"
```
Let's keep the folder name the same as the model name. Will be easy to keep track of folders on the prod server.